robot/Cumulus/tests/browser/affiliations/remove_secondary_affiliation.robot

```diff
@@ -30,7 +30,7 @@ Remove Secondary Affiliation for Contact
     Select Tab                        Related
     Click Related Item Popup Link     Organization Affiliations    ${account}[Name]        Delete
     Wait For Modal                    New                          Affiliation             expected_heading=Delete Affiliation
-    Click Modal Button                Delete
+    Click Element                    //button/span[contains(text(),'Delete')]
```

Comment:
We should add this xpath to the locators file and use the Locator Manager to call it.

Reply:
Made the right change